app/src/@libs/ui/colors/palette.ts

```diff
@@ -0,0 +1,42 @@
+import { sum } from 'lodash';
+
+import { HSLA } from '.';
+
+const paletteColorSaturation = 219;
+const paletteColorLightness = 75;
+
+export const paletteColorsCount = 12;
+
+const paletteHueRanges = [[0, 360]];
```

Comment:
I copied it from my app.
Here's a [YouTube video](https://youtu.be/Muq9gTchz9U) explaining it.